docs/getting_started.md

```diff
@@ -268,7 +269,7 @@ filtered_sce <- filtered_sce[, which(filtered_sce$adt_scpca_filter == "Keep")]
 ```
 
 Alternatively, you can also filter cells out based on your own criteria.
-Quality-control statistics calculated by [`DropletUtils::CleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html) are provided in the `colData` slot of the `altExp` (`colData(altExp(filtered_sce))`), as described in {ref}`Additional SingleCellExperiment components for CITE-seq libraries (with ADT tags) <sce_file_contents:Additional SingleCellExperiment components for CITE-seq libraries (with ADT tags)>`.
+Quality-control statistics calculated by [`DropletUtils::CleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html) (with default parameters) are provided in the `colData` slot of the `altExp` (`colData(altExp(filtered_sce))`), as described in {ref}`Additional SingleCellExperiment components for CITE-seq libraries (with ADT tags) <sce_file_contents:Additional SingleCellExperiment components for CITE-seq libraries (with ADT tags)>`.
```

Comment:
I am not sure if we need this level of detail in the "getting started" document. I would probably just leave this document as it was, reverting both changes.
```suggestion
Quality-control statistics calculated by [`DropletUtils::CleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html) are provided in the `colData` slot of the `altExp` (`colData(altExp(filtered_sce))`), as described in {ref}`Additional SingleCellExperiment components for CITE-seq libraries (with ADT tags) <sce_file_contents:Additional SingleCellExperiment components for CITE-seq libraries (with ADT tags)>`.
```

Reply:
quick clarification here..

>  reverting both changes.

is there a second change/suggestion you meant to leave here ("both")? I wondered if you meant also this change where I added `(with default parameters)` here, but that seems ok to leave in `processing_information`. https://github.com/AlexsLemonade/scpca-docs/blob/259a00069bf156bd32df4b17e20905519b05a0ad/docs/processing_information.md?plain=1#L90 Let me know if there's somewhere else in `getting_started` you were thinking, thanks!